.github/workflows/purge-old-images.yaml

```diff
@@ -0,0 +1,49 @@
+name: Purge old images
+
+permissions:
+  id-token: write
+  contents: read
+  packages: write
+
+on:
+  # Enable manual trigger
+  workflow_dispatch:
+  schedule:
+    # Run twice a day
+    - cron: "30 0,12 * * *"
+
+env:
+  ACTION_LINK: "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+
+jobs:
+  purge_ghcr_dev:
+    name: Delete old unused container images
+    runs-on: ubuntu-latest
+    if: github.repository == 'radius-project/radius'
+    steps:
+      - name: Delete 'dev' containers older than a week
+        uses: snok/container-retention-policy@v3.0.0
+        with:
+          account: radius-project
+          token: ${{ secrets.GH_RAD_CI_BOT_PAT }}
+          image-names: dev/*
+          image-tags: "pr-*"
+          cut-off: 1w
+          keep-n-most-recent: 1
```

Comment:
question: why do we keep 1? if we're cutting off older than a week?

Reply:
I seriously don't have any strong opinions on this one. Not also sure what happens if there are 0 images in a package. What do you think? Any reason we shouldn't keep 1?

Reply:
The package should still exist, just no images. Overall keeping one image version alive should not be a problem.  We can go ahead.